code/game/objects/items/explosives/mine.dm

```diff
@@ -210,30 +282,40 @@
 
 //Note : May not be actual explosion depending on linked method
 /obj/item/explosive/mine/prime()
-	set waitfor = 0
 
 	if(!customizable)
 		create_shrapnel(loc, 12, dir, angle, , cause_data)
 		cell_explosion(loc, 60, 20, EXPLOSION_FALLOFF_SHAPE_LINEAR, dir, cause_data)
+		//so that shrapnel has time to hit mobs before they are knocked over by the explosion
+		addtimer(CALLBACK(GLOBAL_PROC,GLOBAL_PROC_REF(cell_explosion), loc, explosive_power, 20, EXPLOSION_FALLOFF_SHAPE_LINEAR, dir, cause_data), 2.5)
```

Comment:
```suggestion
		addtimer(CALLBACK(GLOBAL_PROC,GLOBAL_PROC_REF(cell_explosion), loc, explosive_power, 20, EXPLOSION_FALLOFF_SHAPE_LINEAR, dir, cause_data), 0.3 SECONDS, numticks = 3)
```

Reply:
2.5 will proc at .5 because numticks is set to 5 default (number of ticks the game checks the action)
decimals like 0.01 SECONDS also don't work